Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ruby:2.3.3
+
+## 1. Image metadata ##
+
+MAINTAINER Tom de Bruijn, tom@tomdebruijn.com
+LABEL version="0.1.0" \
+    description="Image for running the backup Rubygem"
+
+## 2. Add operating system packages ##
+
+# Dependencies for developing and running Backup
+#  * The Nokogiri gem requires libxml2
```

Comment:
Doesn't nokogiri vendor libxml2 nowadays? Might no longer be needed with the upgrade to nokogiri 1.7 #845